syntax = "proto2";

package GameCmd;

enum EnumGameCmd
{
	CMD_BEGIN = 0;
	CMD_APPLY_ENTER_ROOM = 1000;
	CMD_APPLY_QUIT_ROOM = 1001;
	CMD_FLUSH_ROOM = 1002;
	CMD_APPLY_SEAT = 1003;
	CMD_PLAYER_SITDOWN = 1004;
	CMD_APPLY_STANDUP = 1005;
	CMD_PLAYER_STANDUP = 1006;
	CMD_APPLY_FLUSHROOM = 1007;
	CMD_PLAYER_BUYIN = 1008;
    CMD_DEAL_CARDS = 1010; 
	
	CMD_APPLY_FACESHOW = 1011;
	CMD_PLAYER_FACESHOW = 1012;
	CMD_APPLY_CHAT = 1013;
	CMD_PLAYER_CHAT = 1014;
	CMD_APPLY_USE_PROPS = 1015;
	CMD_PLAYER_USE_PROPS = 1016;
	CMD_APPLY_PRESENT_CHIPS = 1017;
	CMD_PLAYER_PRESENT_CHIPS = 1018;
	CMD_APPLY_PRESENT_GIFT = 1019;
	CMD_PLAYER_PRESENT_GIFT = 1020;
	CMD_PLAYER_ENTER_ROOM = 1021;
	CMD_PLAYER_QUIT_ROOM = 1022;
	CMD_APPLY_TRANSMIT_MESSAGE = 1023;
	CMD_PLAYER_TRANSMIT_MESSAGE = 1024;
	
    CMD_APPLY_PRESENT_GIFT_NEWINTER = 1026;
    CMD_PLAYER_PRESENT_GIFT_NEWINTER = 1027;
	CMD_PLAYER_TRUSTEE_ON = 1028;
	CMD_APPLY_TRUSTEE_OFF = 1029;
	CMD_PLAYER_TRUSTEE_OFF = 1030;
	
	CMD_ACTIVE_PLAYER = 1031;   //广播激活玩家
	CMD_APPLY_DIG_CARD = 1032;        //玩家摸牌
	CMD_PLAYER_DIG_CARD = 1033;          //广播玩家摸牌
	CMD_APPLY_DROP = 1034;   //玩家drop
	CMD_PLAYER_DROP = 1035;   //广播玩家drop
	CMD_APPLY_POST_CARD = 1036; //玩家出牌
	CMD_PLAYER_POST_CARD = 1037;//广播玩家出牌
	CMD_APPLY_DECLARE = 1038;  //玩家declare
	CMD_PLAYER_DECLARE = 1039;  //广播玩家declare
	CMD_APPLY_LOSE_DECLARE = 1040;  //输家declare
	CMD_PLAYER_LOSE_DECLARE = 1041;  //广播输家declare
	CMD_GAME_OVER = 1042;      //结算消息
	CMD_APPLY_ROOM_STATUS = 1043; //获取房间状态
	CMD_NOTIFY_WAIT_GAME = 1044; //通知玩家开始或者停止开始倒计时
	CMD_SHUFFLE_CARD = 1045;     //重新洗牌
	CMD_APPLY_CLOSE_ACCOUNT_BOX = 1046;//玩家关闭结算框
	CMD_PLAYER_CLOSE_ACCOUNT_BOX = 1047;//广播玩家关闭结算框
 };

